src/components/LayoutTree.tsx

```diff
@@ -660,6 +661,7 @@ const LayoutTree = () => {
                 // The key must be unique to the thought, both in normal view and context view, in case they are both on screen.
                 // It should not be based on editable values such as Path, value, rank, etc, otherwise moving the thought would make it appear to be a completely new thought to React.
                 key={key}
+                className={classNames({ 'table-col': isTableCol1 })}
```

Comment:
Let's call the class `table-col1` for clarity.

Reply:
Yes.